packages/elements/src/components/MarkdownViewer/CustomComponents/CodeComponent.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { HttpParamStyles, IHttpRequest } from '@stoplight/types';
+
+import { parseHttpRequest } from './CodeComponent';
+
+describe('CodeComponent', () => {
+  describe('parseHttpRequest', () => {
```

Comment:
We don't usually apply the filename scope again as a `describe` block, AFAIK jest recognizes files as an organizational structure anyway.